python/ray/tests/test_placement_group_2.py

```diff
@@ -46,28 +46,18 @@ def value(self):
                 "CPU": 2
             }])
 
-        error_count = 0
-        try:
+        with pytest.raises(ValueError, match="bundle index 3 is invalid"):
             Actor.options(
                 placement_group=placement_group,
                 placement_group_bundle_index=3).remote()
-        except ValueError:
-            error_count = error_count + 1
-        assert error_count == 1
 
-        try:
+        with pytest.raises(ValueError, match="bundle index -2 is invalid"):
             Actor.options(
                 placement_group=placement_group,
                 placement_group_bundle_index=-2).remote()
-        except ValueError:
-            error_count = error_count + 1
-        assert error_count == 2
 
-        try:
+        with pytest.raises(ValueError, match="bundle index must be -1"):
             Actor.options(placement_group_bundle_index=0).remote()
-        except ValueError:
-            error_count = error_count + 1
-        assert error_count == 3
 
```

Comment:
is this some unrelated fixes/improvements to the test?

Reply:
Yes, this is the make the test more debuggable.